pyproject.toml

```diff
@@ -26,43 +26,43 @@ classifiers = [
     "Programming Language :: R"
     ]
 
-requires-python = ">= 3.9"
+requires-python = ">= 3.11"
 
 dependencies = [
```

Comment:
Just a minor comment, but you don't need to specify transitive dependencies -> anndata, jax etc are already included in e.g. scvi-tools. Only if you need to pin them.

Moreover, dependencies such as `pytest` are not runtime dependencies and should be moved into a dev section

Reply:
you're making a lot of good points, but can we open those as issues, as they will otherwise get lost as comments